Vagrantfile

```diff
@@ -146,17 +147,21 @@ Vagrant.configure('2') do |config|
   end
 
   config.vm.define :s_license do |cfg|
-    default_omnibus config
+    default_omnibus(cfg)
+
     cfg.vm.provision :chef_client do |chef|
+      chef.arguments = "--chef-license accept"
```

Comment:
Can we move this into the chef_defaults function instead of copying it around everywhere?

Reply:
why make a separate function instead of adding a line to the existing chef_defaults function?